aemedge/blocks/hero/hero.js

```diff
@@ -22,24 +23,32 @@ function buildAuthorEl(author) {
   return a({ class: 'media-blend__author', href: buildAuthorUrl(author) }, author);
 }
 
+function addAuthorAvatarImg(authorName, avatar) {
+  return getAuthorEntries([authorName]).then((authorEntries) => {
+    if (authorEntries && authorEntries.length > 0) {
+      const ae = authorEntries[0];
+      const picture = new Avatar(ae.title, ae.description, ae.path, ae.image).getImage();
```

Comment:
I'm wondering if it makes sense to build an Avatar to process the image, rather than just calling `createOptimizedPicture`. I understand that nominally our avatar and the udex avatar are related but in practicality they are separate implementations that don't really relate to each other.